static/app/views/settings/project/projectOwnership/index.tsx

```diff
@@ -275,7 +276,9 @@ tags.sku_class:enterprise #enterprise`;
         />
         <IssueOwnerDetails>{this.getDetail()}</IssueOwnerDetails>
 
-        <PermissionAlert />
+        <PermissionAlert
+          access={!editOwnershipeRulesDisabled ? ['project:read'] : ['project:write']}
+        />
```

Comment:
Allow orgs with scope `project:read` to not see the `Permission Alert` modal here if they are allowed to edit Ownership Rules